beluga_amcl/cmake/ament.cmake

```
@@ -59,8 +61,12 @@ ament_target_dependencies(amcl_node_component PUBLIC
   rclcpp_lifecycle
   sensor_msgs
   std_srvs)
-rclcpp_components_register_node(amcl_node_component
-  PLUGIN "beluga_amcl::AmclNode" EXECUTABLE amcl_node)
+rclcpp_components_register_node(
+  amcl_node_component
+  PLUGIN
+  "beluga_amcl::AmclNode"
+  EXECUTABLE
+  amcl_node)
```

@nahueespinosa nit^2: this is an odd formatting. Isn't 
```suggestion
rclcpp_components_register_node(
  amcl_node_component
  PLUGIN "beluga_amcl::AmclNode"
  EXECUTABLE amcl_node)
```
acceptable? Same elsewhere.

---

Fixed in https://github.com/Ekumen-OS/beluga/pull/243/commits/c88191b58b7c0823cefdec729b5e77995ee0d537.